packages/app/obojobo-repository/shared/components/module.scss

```diff
@@ -37,6 +37,11 @@
 		background-color: $color-banner-bg;
 	}
 
+	&.is-new {
+		background-color: lighten($color-action-minor, 15%);
+		font-weight: bold;
```

Comment:
Looks like setting the font-weight here also affects the font-weight of the buttons in the module's menu